### PR TITLE
Feature/panel

### PR DESCRIPTION
**Pull request by dgibson666**
* First cut of Panel component (CSS, no JS needed)
* Default and Sidebar stories
* Removed letter spacing and update line heights with token values in general headings SCSS

### CODE REVIEW

File: packages/vanilla/src/sass/components/_panel.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.cbp-panel {
+  border-style: none;
+  border-radius: var(--cbp-border-radius-soft);
+
+  .cbp-panel-header {
+    color: var(--cbp-color-text-lighter);
+    background-color: var(--cbp-color-gray-cool-50);
+    padding: var(--cbp-space-5x);
+    border-bottom: solid 4px var(--cbp-color-gray-cool-60);
```
Comment (dannyk3941) [inline packages/vanilla/src/sass/components/_panel.scss:9]: Replace 4px with var(--cbp-border-size-xl)

File: packages/vanilla/src/components/panel/panel.stories.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  title: 'Patterns/Panel',
+  argTypes: {
+    tag: {
+      name: 'Semantic Tag',
+      description: 'The semantic tag defining the panel',
+      control: 'radio',
+      options: ['section', 'aside'],
+    },
+    headingLevel: {
+      name: 'Header Text',
```
[the remark this answers is not in the document]
Comment (dgibson666) [reply packages/vanilla/src/components/panel/panel.stories.js:11]: Good catch - copy past error

File: packages/vanilla/src/sass/components/_panel.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    border-style: solid;
+    border-color: var(--cbp-color-gray-cool-10);
+    border-width: var(--cbp-border-size-md);
+    border-top-style: none;
+    border-radius: 0 0 var(--cbp-border-radius-soft) var(--cbp-border-radius-soft);
+  }
+
+  & > :last-child {
+    margin-bottom: 0;
+    padding-bottom: 0;
```
[the remark this answers is not in the document]
Comment (dgibson666) [reply packages/vanilla/src/sass/components/_panel.scss:32]: Good catch. Visually looked right, but it was the wrong nesting context. And & works differently in SASS than LESS...

File: packages/vanilla/src/sass/components/_panel.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
+.cbp-panel {
+  border-style: none;
```
[the remark this answers is not in the document]
Comment (dgibson666) [reply packages/vanilla/src/sass/components/_panel.scss:2]: Yeah, I think I refactored from my first approach and don't need this anymore.

**Review by dannyk3941 — APPROVED** [review]

**Pull request merged**